### PR TITLE
[PLAT-1245] Parser gif bug

### PR DESCRIPTION
**Pull request by preziu**

### CODE REVIEW

File: lib/SlideParser/Parser.php [hunk 1]
```diff
@@ -520,7 +520,12 @@ public function handleImages($html)
 			}
 
 			$path = 'uploads/' . date('Y/m') . '/' . str_random(32) . '.' . $ext;
-			Storage::put('public/' . $path, $data->__toString(), 'public');
+			
+			if (is_string($data)) {
+				Storage::put('public/' . $path, $data, 'public');
+			} else {
+				Storage::put('public/' . $path, $data->__toString(), 'public');
```
Comment (rogatty) [inline lib/SlideParser/Parser.php]: Let's not duplicate the code. How about this instead?
```
$data = is_string($data) ? $data : $data->__toString();
Storage::put('public/' . $path, $data, 'public');
```

Comment (preziu) [reply lib/SlideParser/Parser.php]: sure!

**Comment by rogatty** [comment]
Seems to be obsolete after https://github.com/bethinkpl/wnl-platform/pull/1427/commits/032d613362af44b327b8d663d50127780c099f43

**Comment by gitkuba** [comment]
Fixed in #1427

**Pull request closed without merging**